addons/mail/static/src/models/composer/composer.js

```diff
@@ -287,9 +298,48 @@ function factory(dependencies) {
             body = this._generateMentionsLinks(body);
             body = parseAndTransform(body, addLink);
             body = this._generateEmojisOnHtml(body);
+            return body;
+        }
+
+        /**
+         * Recursive function that process the queue.
+         */
+        async processMessageQueue() {
+            if (!this.isProcessingQueue && !this.isQueueError) {
+                try {
+                    this.update({
+                        isProcessingQueue: true,
+                    });
```

Comment:
outside of `try` (the `try` should ideally only be around the rpc itself and nothing else), so the artificial split with `_postNextMessageInQueued` is kinda strange in this case)